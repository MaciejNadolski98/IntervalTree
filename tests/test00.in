0 10 5
0 7 9 45
0 0 4 37
1 8
1 8
1 0
