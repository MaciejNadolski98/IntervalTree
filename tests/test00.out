45
45
37
